float64 latitude
float64 longitude
int64 fix_code
